Python/MAPL/constants.py

```diff
@@ -50,9 +41,127 @@
 MAPL_SRFPRS  = 98470                  # Pa
 MAPL_KARMAN  = 0.40                   # --
 MAPL_USMIN   = 1.00                   # m/s
-MAPL_AVOGAD  = 6.023E26               # 1/kmol
-
 MAPL_RHO_SEAWATER  = 1026.0          # sea water density [kg/m^3]. SA: should it be = 1026 kg/m^3?
 MAPL_RHO_SEAICE    = 917.0           # sea ice   density [kg/m^3]. SA: should it be = 917  kg/m^3?
 MAPL_RHO_SNOW      = 330.0           # snow density      [kg/m^3]. SA: should it be = 330  kg/m^3?
+MAPL_CELSIUS_TO_KELVIN = 273.15      # K
+
+MAPL_STFBOL  = 5.6734E-8              # W/(m^2 K^4)
+MAPL_AVOGAD  = 6.023E26               # 1/kmol
+MAPL_RUNIV   = 8314.47                # J/(Kmole K)
+
+MAPL_H2OMW   = 18.015                 # kg/Kmole
+MAPL_O3MW    = 47.9982                # kg/Kmole
+MAPL_ALHL    = 2.4665E6               # J/kg @15C
+MAPL_ALHF    = 3.3370E5               # J/kg
+MAPL_ALHS    = MAPL_ALHL+MAPL_ALHF    # J/kg
+
+
+!Internal constants
```

Comment:
```suggestion
#Internal constants
```
The comment character in Python is a hash. Fun with f2py has taught me this a lot!

Reply:
Done. Problems of copy paste lol. Thanks for catching this.